### PR TITLE
App dev add global path

### PR DESCRIPTION
**Pull request by yoanVESSIERE**
Add a new window that follow the main App and we can add global file permission in it

### CODE REVIEW

**Review by mic16 — APPROVED** [review]

File: App/LycaniteApplication.cs [hunk 1]
```diff
@@ -34,18 +38,18 @@ public LycaniteApplication()
         private void Form1_Load(object sender, EventArgs e)
         {
             this.metroSetTabControl1.Padding = new Point(20, 4);
-            if (!this.lycaniteBridge.Connect()) {
+            /*if (!this.lycaniteBridge.Connect()) {
                 this.BeginInvoke(new MethodInvoker(this.Close));
```
Comment (mic16) [inline App/LycaniteApplication.cs:42]: Could you remove these lines?

File: App/LycaniteApplication.cs [hunk 2]
```diff
@@ -34,18 +38,18 @@ public LycaniteApplication()
...
                 int processId = Process.GetCurrentProcess().Id;
                 this.lycaniteBridge.SetLycanitePID((ulong)processId);
                 this.lycaniteBridge.OnLycaniteEvent += this.ProcessLycanite;
-
-                string path = DevicePathMapper.ToDevicePath(WindowsPath);
-                if (path != null) {
-                    this.lycaniteBridge.SetGlobalFilePermissions(path, ELycanitePerm.LYCANITE_READ);
-                }
-            }
+            }*/
```
Comment (Shiranuit) [inline App/LycaniteApplication.cs:47]: This comment should be removed, without this it would be impossible to communicate with the driver

**Review by Shiranuit — CHANGES_REQUESTED** [review]

File: App/LycaniteApplication.cs (hunk 1 above)
Comment (yoanVESSIERE) [reply App/LycaniteApplication.cs]: Done

File: App/LycaniteApplication.cs (hunk 2 above)
Comment (yoanVESSIERE) [reply App/LycaniteApplication.cs]: Done

**Review by Shiranuit — DISMISSED** [review]

**Review by Vaati3 — DISMISSED** [review]

**Review by cyrilbegard — DISMISSED** [review]

**Review by AntoineMaillard06 — APPROVED** [review]

**Review by Vaati3 — APPROVED** [review]

**Comment by Shiranuit** [comment]
image

**Review by mic16 — APPROVED** [review]

**Review by Shiranuit — APPROVED** [review]

**Review by theocerutti — APPROVED** [review]

**Review by MathildeCouston — APPROVED** [review]

**Pull request merged**